packages/fxa-auth-server/test/local/routes/recovery-keys.js

```diff
@@ -112,6 +120,117 @@ describe('POST /recoveryKey', () => {
     });
   });
 
+  describe('should change account recovery key when explicitely requested', () => {
```

Comment:
The logic in the recovery-key route is just complex enough to cause problems, and there's error handling to control the flow. I Feel like there might be a couple more scenarios to cover:

When the the enbabled is false
When replace key is false 
When the key already exists

-

Reply:
Thanks for spotting this missing test case! 

I reviewed the logic in the recovery-key route and added a test case for the scenario where an enabled key already exists, but replaceKey is false.